[2022-11-02 14:48:30,498] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:48:15.664734+00:00 [queued]>
[2022-11-02 14:48:30,525] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:48:15.664734+00:00 [queued]>
[2022-11-02 14:48:30,526] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-02 14:48:30,528] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-11-02 14:48:30,529] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-02 14:48:30,573] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_xetra_etl> on 2022-11-02 14:48:15.664734+00:00
[2022-11-02 14:48:30,583] {standard_task_runner.py:52} INFO - Started process 495 to run task
[2022-11-02 14:48:30,591] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xetra_DAG', 'run_xetra_etl', 'manual__2022-11-02T14:48:15.664734+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/xetra_DAG.py', '--cfg-path', '/tmp/tmp4z7il8ku', '--error-file', '/tmp/tmpa5gjjbq2']
[2022-11-02 14:48:30,593] {standard_task_runner.py:80} INFO - Job 140: Subtask run_xetra_etl
[2022-11-02 14:48:30,746] {task_command.py:369} INFO - Running <TaskInstance: xetra_DAG.run_xetra_etl manual__2022-11-02T14:48:15.664734+00:00 [running]> on host 0163e0790208
[2022-11-02 14:48:30,928] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xetra_DAG
AIRFLOW_CTX_TASK_ID=run_xetra_etl
AIRFLOW_CTX_EXECUTION_DATE=2022-11-02T14:48:15.664734+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-02T14:48:15.664734+00:00
[2022-11-02 14:48:30,931] {logging_mixin.py:115} INFO - /opt/***/meta_file.csv
[2022-11-02 14:48:46,696] {python.py:173} INFO - Done. Returned value was: None
[2022-11-02 14:48:50,042] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xetra_DAG, task_id=run_xetra_etl, execution_date=20221102T144815, start_date=20221102T144830, end_date=20221102T144850
[2022-11-02 14:48:57,682] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-02 14:48:57,752] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
